pkg/driver/controller.go

```diff
@@ -263,12 +262,13 @@ func CreateLVMVolume(ctx context.Context, req *csi.CreateVolumeRequest,
 	}
 
 	owner := selected[0]
-	klog.Infof("scheduling the volume %s/%s on node %s", params.VolumeGroup, volName, owner)
+	klog.Infof("scheduling the volume %s/%s on node %s",
+		params.VgRegex.String(), volName, owner)
 
 	volObj, err := volbuilder.NewBuilder().
 		WithName(volName).
 		WithCapacity(capacity).
-		WithVolGroup(params.VolumeGroup).
+		WithVolGroupRegex(params.VgRegex.String()).
```

Comment:
this should not work, can you try the storage class without volgroup? Have you tested it with the volgroupregx only in the storageclass?